src/LaravelVuePagination.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-    <renderless-laravel-vue-pagination :data="data" :limit="limit" v-on:pagination-change-page="onPaginationChangePage">
+    <renderless-laravel-vue-pagination :data="data" :limit="limit" :show-disabled="showDisabled" v-on:pagination-change-page="onPaginationChangePage">
         <ul class="pagination" v-if="computed.total > computed.perPage" slot-scope="{ data, limit, computed, prevButtonEvents, nextButtonEvents, pageButtonEvents }">
-            <li class="page-item pagination-prev-nav" v-if="computed.prevPageUrl">
+            <li class="page-item pagination-prev-nav" :class="{'disabled': !computed.prevPageUrl}" v-if="computed.prevPageUrl || showDisabled">
```

Comment:
Instead of using the `disabled` class here, we should probably use the `disabled` prop instead. (e.g. `:disabled="!computed.prevPageUrl"`). Same for below.

Reply:
Applying the class because it is used by bootstrap to style the disabled link and apply pointer-events:none to it. 

See: http://getbootstrap.com/docs/4.1/components/pagination/

I’m thinking it would probably be good to add tabinex=“-1” though.

Reply:
Ah fair enough 👍 Just need to add a test then.